### PR TITLE
130 multiple column names

### LINKED ISSUE #130: Add multiple column names support
Could you add support of multiple column names please?

 **Example**:

Class:

```
public class Employee {

    @ExcelCellName(name="id,identifier")
    private long employeeId;

    @ExcelCellName(name="name,title,nick", delimeter=",")
    private String name;

    @Override
    public String toString() {
        return "Employee{" +
                "employeeId=" + employeeId +
                ", name='" + name + '\'' +
                '}';
    }

}
```
_Writing:_

Result excel table:

| id,identifier    | name   |
|--------|--------|
| 123923 | Joe    |
| 123123 | Sophie |
| 135923 | Paul   |

_Reading:_

Source excel table:

| id,identifier    | nick   |
|--------|--------|
| 123923 | Joe    |
| 123123 | Sophie |
| 135923 | Paul   |

Result:

Employee{employeeId=123923, name='Joe'}

### PR DESCRIPTION
**Pull request by vaa25**
resolves #130 

merge after #129 

@ozlerhakan you can use ConcurrencyTest to be sure that any performance issues absent there.

### CODE REVIEW

File: src/main/java/com/poiji/bind/mapping/PoijiHandler.java [hunk 1]
```diff
@@ -185,6 +180,23 @@ private boolean setValue(Field field, int column, String content, Object ins) {
...
 
+    private Integer getFieldColumnFromExcelCellName(final ExcelCellName excelCellName) {
+        final String titleName = options.getCaseInsensitive()
+            ? excelCellName.value().toLowerCase()
+            : excelCellName.value();
+        if (excelCellName.delimeter().isEmpty()) {
+            return columnIndexPerTitle.get(titleName);
+        } else {
+            final String[] possibleTitles = titleName.split(excelCellName.delimeter());
+            for (final String possibleTitle : possibleTitles) {
```
Comment (ozlerhakan) [inline src/main/java/com/poiji/bind/mapping/PoijiHandler.java]: thank so much you for you effort @vaa25 ,  but this is the case I care about more. So we don't need a if-else statement + a for loop. Thats why it leads a performance for huge excel files.

Comment (vaa25) [reply src/main/java/com/poiji/bind/mapping/PoijiHandler.java]: @ozlerhakan did you run ConcurrencyTest? What performance penalty did you catch for huge file?

Comment (vaa25) [reply src/main/java/com/poiji/bind/mapping/PoijiHandler.java]: @ozlerhakan review again please.

Comment (vaa25) [reply src/main/java/com/poiji/bind/mapping/PoijiHandler.java]: @ozlerhakan read performance increased for 30%

**Comment by vaa25** [comment]
moved to https://github.com/vaa25/poiji

**Pull request closed without merging**